pkg/webui/console/views/app/index.js

```diff
@@ -43,31 +43,32 @@ export default class ConsoleApp extends React.Component {
     if (pageData && pageData.error) {
       return <FullViewError error={pageData.error} />
     }
-
     return (
-      <ErrorView ErrorComponent={FullViewError}>
-        <div className={style.app}>
-          <Helmet
-            titleTemplate="%s - Console - TTN Stack"
-            defaultTitle="TTN Stack Console"
-          />
-          <div id="modal-container" />
-          <Header className={style.header} />
-          <main className={style.main}>
-            <div>
-              <SideNavigation />
-            </div>
-            <div className={style.content}>
-              <Switch>
-                {/* routes for registration, privacy policy, other public pages */}
-                <Route path={`${env.app_root}/login`} component={Login} />
-                <Route path={env.app_root} component={Landing} />
-              </Switch>
-            </div>
-          </main>
-          <Footer className={style.footer} />
-        </div>
-      </ErrorView>
+      <EnvProvider env={env}>
+        <ErrorView ErrorComponent={FullViewError}>
+          <div className={style.app}>
+            <IntlHelmet
+              titleTemplate={`%s - ${env.site_title} - ${env.site_name}`}
```

Comment:
Let's account for the case of the site title not being set:
```suggestion
              titleTemplate={`%s - ${env.site_title ? `${env.site_title} -` : ''}${env.site_name}`}
```